README.rst

```diff
@@ -37,6 +37,10 @@ Set up your virtualenv with your preferred method. For me, using
    $ pip install pip-tools
    $ pip-sync requirements/dev.txt requirements/base.txt
 
+You'll have to read the contents of your ``.env`` file into your local
+shell's environment. You may want to set up a system to do this each
+time you activate this virtualenv.
+
```

Comment:
Links to prior art would help, here. "You'll need to do this thing" on its own is kinda contributor-hostile, even if it is inadvertent.

Reply:
True. I'll ponder how to write that up.

Reply:
Don't hold this PR up on it, unless there's a clear answer.